CHANGELOG.md

```diff
@@ -0,0 +1,178 @@
+# Changelog
+
+## [v1.1](https://github.com/arborx/arborx/tree/v1.1) (2021-09-23)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.0...v1.1)
+
+**New features:**
+
+- Allow k-DOP as a bounding volume in BVH [\#386](https://github.com/arborx/ArborX/pull/386)
+
+**Enhancements:**
+
+- Store a dimension value in a DBSCAN datafile [\#523](https://github.com/arborx/ArborX/pull/523)
+- Provide computation of a Sphere-Box distance [\#495](https://github.com/arborx/ArborX/pull/495)
+- Provide Box-Box distance for the Nearest query [\#494](https://github.com/arborx/ArborX/pull/494)
+- Use double to normalize Ray direction [\#535](https://github.com/arborx/ArborX/pull/535)
+- Improve performance of the brute force algorithm [\#534](https://github.com/arborx/ArborX/pull/534)
+- Improve performance of the DBSCAN algorithm (new FDBSCAN-DenseBox algorithm) [\#508](https://github.com/arborx/ArborX/pull/508)
+- Allow non-View input for DBSCAN [\#509](https://github.com/arborx/ArborX/pull/509)
+- Install ArborXConfigVersion.cmake [\#521](https://github.com/arborx/ArborX/pull/521)
+- Add a simple intersection example [\#513](https://github.com/arborx/ArborX/pull/513)
+- Improve performance for the SYCL backend through the use of oneDPL for sorting [\#456](https://github.com/arborx/ArborX/pull/456)
+
+**Fixed bugs:**
+
+- Fix a bug in DBSCAN noise marking [\#525](https://github.com/arborx/ArborX/pull/525)
+
+## [v1.0](https://github.com/arborx/arborx/tree/v1.0) (2021-03-13)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.0-rc0...v1.0)
+
+**New features:**
+
+- Allow early termination of a traversal by a thread [\#427](https://github.com/arborx/ArborX/pull/427)
+- Implement DBSCAN clustering algorithm [\#331](https://github.com/arborx/ArborX/pull/331)
+- Implement brute-force algorithm [\#468](https://github.com/arborx/ArborX/pull/468)
+- Implement simple ray-tracing [\#414](https://github.com/arborx/ArborX/pull/414)
```

Comment:
```suggestion
- Add ray-tracing example [\#414](https://github.com/arborx/ArborX/pull/414)
```

Reply:
We did not simply add an example, we also added `Ray` class and intersections with it. I would like to revert this change.

Reply:
Everything is covered by our "adding initial ray tracing support"

Reply:
I don't see "Adding initial ray tracing support" anywhere in the changelog.